gamemodes/cinema/gamemode/misc/cl_hl2hud.lua

```diff
@@ -49,6 +49,12 @@ function GM:PlayerStartVoice(ply)
     end
 end
 
+function GM:DrawDeathNotice(x, y)
```

Comment:
Just use a hook

Reply:
the death notice is already drawn by the time it gets to the hook I think, even just having it only contain "return" doesn't stop the death notices

Reply:
Try returning true. Returning nothing is nil (same as not having a return statement at all). You have to return something non-nil to "block" a hook. It looks to me like the death notice drawer is a gamemode function, so it would run only if any hook.Add functions don't block it.